extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -632,3 +633,158 @@ descendant_tag_list: {
   tag: "VAR"
   tag: "WBR"
 }
+tags: {  # <amp-story-page-attachment>
+  html_format: AMP
+  tag_name: "AMP-STORY-PAGE-ATTACHMENT"
+  mandatory_ancestor: "AMP-STORY-PAGE"
+  descendant_tag_list: "amp-story-page-attachment-allowed-descendants"
+  mandatory_last_child: true
+  attrs: {
+    name: "layout"
+    mandatory: true
+    value: "nodisplay"
+  }
+}
+descendant_tag_list {
+  name: "amp-story-page-attachment-allowed-descendants"
+  tag: "A"
+  tag: "ABBR"
+  tag: "ADDRESS"
+  tag: "AMP-3D-GLTF"
```

Comment:
This might be hard to maintain. What is the criteria for deciding what's in this list? Maybe leave a comment in here explaining that at the minimum so that the next person can use it as a guide for reasonable decisions.

Reply:
We tried to keep it as open as possible, only restricting extensions that would hurt the UX (ie: lightboxes like amp-lightbox or others would create a modal within a modal), or create potential security issues (ie: forms).
I added that as a comment

Reply:
One core principle is whitelist vs. blacklist: if a new `amp-foo` component is created, in the time between when it is created and when we vet its use for stories, should it be allowed, or disallowed?

For us, we want to disallow, to retain control over the user experience and make sure that all allowed components fit with three user experience we intend to enable.